### PR TITLE
New package: urxvt-font-size-1.3

### PR DESCRIPTION
**Pull request by tommyjanna**

### CODE REVIEW

File: srcpkgs/urxvt-font-size/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'urxvt-font-size'
+pkgname=urxvt-font-size
+version=1.3
+revision=1
+archs="i686 x86_64"
```
Comment (ericonr) [inline srcpkgs/urxvt-font-size/template:5]: Why is this set?

Comment (tommyjanna) [reply srcpkgs/urxvt-font-size/template]: Wasn't entirely sure if required... I now see it's unnecessary from the packaging manual.

Amended the original commit.

**Comment by ericonr** [comment]
That said, given that in their [README](https://github.com/majutsushi/urxvt-font-size/blob/master/README.markdown) upstream recommends installing this in `$HOME`, and that this is mostly a personal config, I'm not sure it fits within our [quality requirements(https://github.com/void-linux/void-packages/blob/master/Manual.md#quality_requirements) :/

**Comment by ahesford** [comment]
I agree with @ericonr, this belongs in `$HOME` rather than as a package.

**Pull request closed without merging**